lms/static/styles/components/_LMSGrader.scss

```diff
@@ -0,0 +1,35 @@
+.LMSGrader {
+  font-size: 18px;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  color: white;
+  h2, h3 {
+    margin-left: 20px;
+  }
+  ul {
```

Comment:
This will generate a `.LMSGrader ul` selector which will apply to any list rendered anywhere within the grader, including its child components. This means that a change here could unexpectedly affect a list rendered by some other component. Our convention is to use class selectors only in most cases to avoid these hazards.

Reply:
oops, this was an oversite! I forgot it was a root class like this